styles/globals.css

```diff
@@ -1,26 +1,27 @@
-/*! purgecss start ignore */
-@import "tailwindcss/base";
-@import "tailwindcss/components";
-@import url(https://cdn.jsdelivr.net/gh/tonsky/FiraCode@4/distr/fira_code.css);
-@import "swiper/swiper.min";
-@import "swiper/modules/a11y/a11y.min";
-@import "swiper/modules/navigation/navigation.min";
-/*! purgecss end ignore */
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+@tailwind variants;
+
+@import 'swiper/css';
```

Comment:
@magicmatatjahu I changed the swiper css libraries here because they are giving some import errors on present version of node v16. Is this correct?

Reply:
For swiper I bumped NodejS version to v16 and I didn't see any errors. I will check that in my console.

Reply:
If you head over to this commit https://github.com/asyncapi/website/pull/804/commits/4ad7465e0db33a1a8d028d3179651078dc6ea5c5 in this PR, I got the error that it can't extract `swiper/swiper.min` inside the `swiper` package. Hence, when I saw the documentation on the website of swiper, I got these CSS files there and added them to the global.css.

Reply:
In previous PR (newsroom) it didn't see this `swiper.css` and now it does 😆 Ok, it's resolved.